### ISSUE TITLE
[Story] Warn if all other users are blocked in the DM/room

### ISSUE THREAD

**Issue by kittykat** [vector-im/element-meta#1786]
### A story should take roughly a week or a sprint to finish. Each story is usually made up of a number of tasks that take half to a full day.

As a user,
I want to know when I have blocked everyone in a room
so that I know that no one will receive my messages when I send messages into that room

This if follow on from #1106

## Scope
```[tasklist]
### Tasklist
- [ ] In a DM, if I have blocked the other user, I get a warning when I try to send a message when I try to send a message to the room
- [ ] In a group chat, if I have blocked all other users in the room, I get a warning when I try to send a message to the room
```

**Android**
- [ ] QA signoff on completion
- [ ] Design signoff on completion
- [ ] Product signoff on completion

**iOS**
- [ ] QA signoff on completion
- [ ] Design signoff on completion
- [ ] Product signoff on completion

##  Stretch goals
None at this time
<or add a tasklist>

## Out of scope
-

**Comment by manuroe** [comment]
@VolkerJunginger should we work on this now?

It has been discovered during QA on https://github.com/vector-im/element-meta/issues/1106. Look for  "You won’t be able to receive or send any messages until you unblock them" in this issue.
This is the last action of the https://github.com/vector-im/internal-planning/issues/334 epic.

**Comment by VolkerJunginger** [comment]
No, let's put this into the backlog. This is neither urgent nor has big impact.

**Comment by manuroe** [comment]
I added it to the backog
